tests/types/gradient/test.typ

```diff
@@ -0,0 +1,10 @@
+#import "/src/lib.typ" as z
+#import "/tests/utility.typ": *
+
+#show: show-rule.with();
+
+#let schema = z.gradient()
+
+= types/gradient
+== Input types
+#z.parse(gradient.linear(..color.map.rainbow), schema)
```

Comment:
If we only care about the validation part of the test we may discard the result of `parse`, this avoids unnecessary repository bloat from large test images.

It's partially my fault, as I haven't added compile only tests to typst-test yet.